### PR TITLE
Appointment class updated, updated fields

### PR DESCRIPTION
**Pull request by Mogrix**

### CODE REVIEW

File: appointmentScheduler/src/main/java/com/appointment/model/Appointment.java [hunk 1]
```diff
@@ -1,4 +1,44 @@
...
+    }
+
+    public void setStartTime(LocalDateTime startTime) {
+        this.startTime = startTime;
+    }
+
+    public LocalDateTime getStartTime() {
+        return startTime;
+    }
+
```
Comment (romanpetras) [inline appointmentScheduler/src/main/java/com/appointment/model/Appointment.java:43]: remove empty line

File: appointmentScheduler/pom.xml [hunk 2]
```diff
@@ -8,5 +8,9 @@
     <artifactId>appointmentScheduler</artifactId>
     <version>1.0-SNAPSHOT</version>
 
+    <properties>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+    </properties>
```
Comment (romanpetras) [inline appointmentScheduler/pom.xml:15]: there should be no empty lines between, consecutively closing, tags or parenthesis... please, remove remove empty line

File: appointmentScheduler/appointmentScheduler.iml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module org.jetbrains.idea.maven.project.MavenProjectsManager.isMavenModule="true" type="JAVA_MODULE" version="4">
```
Comment (romanpetras) [inline appointmentScheduler/appointmentScheduler.iml:2]: please, add this file to .gitignore

File: .idea/compiler.xml [hunk 4]
```diff
@@ -10,7 +10,7 @@
       </profile>
     </annotationProcessing>
```
Comment (romanpetras) [inline .idea/compiler.xml:11]: please, add this file to .gitignore

File: appointmentScheduler/pom.xml (hunk 2 above)
Comment (Mogrix) [reply appointmentScheduler/pom.xml:15]: there are no empty lines between consecutively closing tags or parentheses

Comment (romanpetras) [reply appointmentScheduler/pom.xml:15]: please, pay attention to line 15 which is an empty line between  </properties> and </project> closing tags

Comment (romanpetras) [reply appointmentScheduler/pom.xml:15]: properties and project closing tags

**Pull request merged**